src/pages/test-report/test-report.scss

```diff
@@ -1,3 +1,23 @@
 page-test-report {
+  .content-ios {
```

Comment:
Does this work in browser?

Reply:
Ionic is setup to use the ios styles in the browser as its default instead of material ui which is the default. So all the ios specific styles are shown.

Code which does this is in the app.module `IonicModule.forRoot(App, { mode: 'ios' }),`